modules/tensor_mechanics/src/materials/ComputeStrainBase.C

```diff
@@ -56,11 +56,17 @@ ComputeStrainBase::ComputeStrainBase(const InputParameters & parameters)
     _eigenstrains[i] = &getMaterialProperty<RankTwoTensor>(_eigenstrain_names[i]);
   }
 
-  // Checking for consistency between mesh size and length of the provided displacements vector
-  if (_ndisp != _mesh.dimension())
-    mooseError(
-        "The number of variables supplied in 'displacements' must match the mesh dimension.");
+  if (_ndisp == 1 && _volumetric_locking_correction)
+    mooseError("Volumetric locking correction has to be set to false for 1-D problems.");
```

Comment:
paramError()

Reply:
I can change this but this is only showing up as added by me because I removed something else in the constructor this error already existed as is.  Same as the one for use_displaced_mesh below.

Reply:
Changed